lib/asset_cloud/base.rb

```diff
@@ -8,8 +8,7 @@ class IllegalPath < StandardError
 
   class Base
     cattr_accessor :logger
-
-    VALID_PATHS = /\A[a-z0-9][a-z0-9_\-\/]+([a-z0-9][\w\-\ \.\@]*\.\w{2,6})?\z/i
+    VALID_PATHS = /\A(([a-z0-9])|((\.?[\w\-\@])([\w\-\@]|[\ ][\w\-\@]|[\/][\w\-\@\.]|[\.][\w\-\@]+)*))\z/i
```

Comment:
I also agree this improves what was there in terms of allowable output, so if you guys want to proceed I'm okay with that.